lib/main.coffee

```diff
@@ -188,6 +188,32 @@ module.exports =
         return p
     return execPath
 
+  getProjectPath: (lintedFile) ->
```

Comment:
This whole function can be replaced with:
```js
atom.project.relativizePath(lintedFile)[0]
```

See [here](https://atom.io/docs/api/v1.8.0/Project#instance-relativizePath) for the documentation.